misc/acrn-config/board_config/board_info_h.py

```diff
@@ -66,6 +67,18 @@ def find_hi_mmio_window(config):
     print("#define HI_MMIO_SIZE\t\t\t{}UL".format(hex(board_cfg_lib.HI_MMIO_OFFSET)), file=config)
 
 
+def find_p2sb_bar_addr():
+    iomem_lines = board_cfg_lib.get_info(common.BOARD_INFO_FILE, "<IOMEM_INFO>", "</IOMEM_INFO>")
+
+    for line in iomem_lines:
+        if 'INTC1020:' in line:
```

Comment:
INTC1020 is specific for ehl right? better add condition here that board is ehl_crb_b